crates/router/src/core/errors/api_error_response.rs

```diff
@@ -156,7 +156,7 @@ pub enum ApiErrorResponse {
     #[error(error_type = ErrorType::ObjectNotFound, code = "HE_04", message = "Address does not exist in our records")]
     AddressNotFound,
     #[error(error_type = ErrorType::ObjectNotFound, code = "HE_04", message = "Card with the provided bin does not exist")]
-    CardBinNotFound,
+    InvalidCardIIN,
```

Comment:
```suggestion
    InvalidCardIin,
```

Let's keep this consistent with the Rust API guidelines.

From https://rust-lang.github.io/api-guidelines/naming.html#casing-conforms-to-rfc-430-c-case:

> In `UpperCamelCase`, acronyms and contractions of compound words count as one word: use `Uuid` rather than `UUID`, `Usize` rather than `USize` or `Stdin` rather than `StdIn`.